app/adopt_a_paw/page.tsx

```diff
@@ -33,47 +33,29 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 const AdoptAPaw = () => {
   const dispatch = useAppDispatch();
 
-  const petsStateSelector = useAppSelector((state) => ({
-    pets: state.pets.pets,
-    selectedPet: state.selectedPet.selectedPet,
-    formSelectedPet: state.formSelectedPet.formSelectedPet,
-    visiblePets: state.visiblePets.visiblePets,
-  }));
+  const pets = useAppSelector((state) => state.pets.pets);
```

Comment:
Hi @Daniel54088 ,

This snippet is about recovering from the usage of CombineSeletor. The former implementation caused a re-rendering issue, leading to test case failures.

I've been stuck on understanding the correct usage of combineSelector for a day and couldn't find sufficient references.

The only resource I could find was this one, but it's outdated: https://heygrady.github.io/redux-selectors/ 

Could you please provide any suggestions? Thanks